schemas/ispyb/updates/2023_02_14_SSX_experiment.sql

```diff
@@ -0,0 +1,148 @@
+        'Mesh',
+        'Burn',
+        'MAD - Inverse Beam',
+        'Characterization',
+        'Dehydration',
+        'tomo',
+        'experiment',
+        'EM',
+        'PDF',
+        'PDF+Bragg',
+        'Bragg',
+        'single particle',
+        'Serial Fixed',
+        'Serial Jet',
+        'Standard',
+        'Time Resolved',
+        'Diamond Anvil High Pressure',
+        'Custom',
+        'XRF map',
+        'Energy scan',
+        'XRF spectrum',
+        'XRF map xas',
+        'Mesh3D',
+        'Screening',
+        'Still',
+        'SSX-Chip',
+        'SSX-Jet'
+      ) 
+      DEFAULT NULL COMMENT 'Standard: Routine structure determination experiment. Time Resolved: Investigate the change of a system over time. Custom: Special or non-standard data collection.';
+
+INSERT INTO ExperimentType (experimentTypeId, name, proposalType, active)
+VALUES 
+  (38, 'Still','mx', 1),
+  (39, 'SSX-Chip','mx', 1),
+  (40, 'SSX-Jet','mx', 1);
+
+ALTER TABLE
+    `ComponentType`
+ADD
+    UNIQUE (`name`);
+
+INSERT INTO
+    ComponentType (componentTypeId, name)
+VALUES
+    (5, 'Ligand'),
+    (6, 'Buffer'),
+    (7, 'JetMaterial');
+
+CREATE TABLE `Component` (
+    `componentId` int(11) unsigned PRIMARY KEY AUTO_INCREMENT,
+    `componentTypeId` int(11) unsigned NOT NULL,
+    `proposalId` int(10) unsigned DEFAULT NULL,
+    `name` varchar (255) NOT NULL,
+    `composition` varchar(255) DEFAULT NULL,
+    FOREIGN KEY (`componentTypeId`) REFERENCES `ComponentType`(`componentTypeId`),
+    FOREIGN KEY (`proposalId`) REFERENCES `Proposal`(`proposalId`) ON DELETE CASCADE ON UPDATE CASCADE
+) DEFAULT CHARSET = utf8mb4 COMMENT = 'Description of a component that can be used inside a crystal or a sample.';
+
+CREATE TABLE `SampleComposition` (
+    `sampleCompositionId` int(11) unsigned PRIMARY KEY AUTO_INCREMENT,
+    `componentId` int(11) unsigned NOT NULL,
+    `blSampleId` int(11) unsigned NOT NULL,
+    `concentrationTypeId` int(11) unsigned DEFAULT NULL,
+    `abundance` float DEFAULT NULL COMMENT 'Abundance or oncentration in the unit defined by concentrationTypeId.',
+    `ratio` float DEFAULT NULL,
+    `ph` float DEFAULT NULL,
+    FOREIGN KEY (`componentId`) REFERENCES `Component`(`componentId`),
+    FOREIGN KEY (`blSampleId`) REFERENCES `BLSample`(`blSampleId`),
+    FOREIGN KEY (`concentrationTypeId`) REFERENCES `ConcentrationType`(`concentrationTypeId`)
+) DEFAULT CHARSET = utf8mb4 COMMENT = "Links a sample to it's components with a specified abundance or ratio.";
+
+CREATE TABLE `CrystalComposition` (
+    `crystalCompositionId` int(11) unsigned PRIMARY KEY AUTO_INCREMENT,
+    `componentId` int(11) unsigned NOT NULL,
+    `crystalId` int(11) unsigned NOT NULL,
+    `concentrationTypeId` int(10) unsigned DEFAULT NULL,
+    `abundance` float DEFAULT NULL COMMENT 'Abundance or oncentration in the unit defined by concentrationTypeId.',
+    `ratio` float DEFAULT NULL,
+    `ph` float DEFAULT NULL,
```

Comment:
I presume this is `pH` - should the capitalisation reflect common usage to make its meaning clearer?